src/blocktypes/containers.nix

```diff
@@ -22,7 +23,38 @@
       fragment,
       fragmentRelPath,
       target,
-    }: [
+    }: let
```

Comment:
I think we overlooked:

`print-image` action (in case somebody uses it). Could be plural, now...

And also the docstring needs update.

Reply:
should we just stick to printing the first tag or print them all in a loop?

Reply:
I would almost vote for "all in a loop".

It would be a breaking change, but then, this action's purpose is utterly informational. So all the tags seems like relevant information.

People are unlikely to use that for scripting (or so I hope!!). That's what the other actions are for, after all.

Reply:
The list of tags in the meta would be more suited for scripting purposes anyway